VideoLocker/src/main/java/org/edx/mobile/module/analytics/ISegmentImpl.java

```diff
@@ -540,21 +540,39 @@ public Properties trackUserLogout() {
     }
 
     /**
-     * This function is used send Screen View Details
-     * @param screenName
-     * @return 
+     * This function is used to send Screen View Details
+     *
+     * @param screenName The screen name to track
+     * @return
```

Comment:
This should describe what is being returned. I see that it's empty in all methods, so it could be changed for them all.